static/generated/privacy.pug

```diff
@@ -117,17 +121,14 @@ html(lang="en")
     h4 Cookies
 
     p
-      | Separately, Compiler Explorer uses small pieces of information stored on your computer: Cookies and Browser Local
-      | Storage. Cookies are only used with the user's permission, and are used with external analytics services (e.g.
-      | Google Analytics) to gather statistics on Compiler Explorer usage. This information is used to help the Compiler
-      | Explorer team plan for future updates and hardware upgrades in order to ensure the site remains stable and
-      | responsive. Local storage is used to remember the user's settings, source code and user interface configuration, so
+      | Compiler Explorer uses small pieces of information stored on your computer: Browser Local
```

Comment:
Maybe we need to put something about the sentry cookies here too (there's no mention of Cookies now in this `h4` of "Cookies").

```suggestion
      | Compiler Explorer uses small pieces of information stored on your computer: Cookies and Browser Local
```

Reply:
maybe need to reinstate the line about "Cookies are only used with the user's permission"